q2_diversity/_alpha/_method.py

```diff
@@ -22,7 +22,7 @@ def non_phylogenetic_metrics():
             'dominance', 'doubles', 'enspie', 'esty_ci', 'fisher_alpha',
             'goods_coverage', 'heip_e', 'kempton_taylor_q', 'margalef',
             'mcintosh_d', 'mcintosh_e', 'menhinick', 'michaelis_menten_fit',
-            'observed_otus', 'osd', 'pielou_e', 'robbins', 'shannon',
+            'feature_counts', 'osd', 'pielou_e', 'robbins', 'shannon',
```

Comment:
@gregcaporaso and I chatted about this metric rename. We'd prefer to leave it as `observed_otus` because the name has already changed (in QIIME 1/PyCogent, it used to be `observed_species`, then changed to `observed_otus` somewhere around QIIME 1.8). Using the current name also lets us link to the skbio diversity metric docs since the names correspond 1-1.

We discussed making an alias but then users might be confused about how the two metrics compare. Right now we don't have a great way to document each metric so it seems safest/easiest on users if we keep the current (less-than-ideal) name. If we do change it in the future, we'd probably call it `observed_features` to keep the naming scheme intact.